src/modules/flow/settings.c

```diff
@@ -195,9 +195,45 @@ void global_data_reset_param_defaults(void){
 	strcpy(global_data.param_name[PARAM_BOTTOM_FLOW_WEIGHT_NEW], "BFLOW_W_NEW");
 	global_data.param_access[PARAM_BOTTOM_FLOW_WEIGHT_NEW] = READ_WRITE;
 
-	global_data.param[PARAM_BOTTOM_FLOW_SERIAL_THROTTLE_FACTOR] = 10.0f;
-	strcpy(global_data.param_name[PARAM_BOTTOM_FLOW_SERIAL_THROTTLE_FACTOR], "BFLOW_THROTT");
-	global_data.param_access[PARAM_BOTTOM_FLOW_SERIAL_THROTTLE_FACTOR] = READ_WRITE;
+	global_data.param[PARAM_BOTTOM_FLOW_PUB_RATE] = 10.0f;
+	strcpy(global_data.param_name[PARAM_BOTTOM_FLOW_PUB_RATE], "BFLOW_RATE");
+	global_data.param_access[PARAM_BOTTOM_FLOW_PUB_RATE] = READ_WRITE;
+
+	global_data.param[PARAM_EXPOSURE_MAX] = 500;
```

Comment:
Increasing this value helps in low light, but it also makes the max velocity you can go lower. If you have trouble with velocity maxing out, lower this to around 200 or 100.